docs/reference/installation.rst

```diff
@@ -8,7 +8,7 @@ Installation
 Prerequisites
 -------------
 
-PHP ^7.2 and Symfony ^4.4 are needed to make this bundle work, there are
+PHP ^7.3 and Symfony ^4.4 are needed to make this bundle work, there are
```

Comment:
IMHO we should remove the version information from out docs. Composer will solve this problem

Reply:
I will revert this change, in any case it could be done first on 3.x, since the version constraints are the same.

(I also agree that we could remove the versions from docs because they always get outdated).

Reply:
Done. I will move this conversation to the docs update issue on this bundle.